llvm/lib/CodeGen/SelectionDAG/InstrEmitter.cpp

```diff
@@ -1412,6 +1412,13 @@ EmitSpecialNode(SDNode *Node, bool IsClone, bool IsCloned,
       }
     }
 
+    // Add rounding control registers as implicit def for inline asm.
+    if (MF->getFunction().hasFnAttribute(Attribute::StrictFP)) {
+      ArrayRef<MCPhysReg> RCRegs = TLI->getRoundingControlRegisters(AsmStr);
+      for (MCPhysReg Reg : RCRegs)
+        MIB.addReg(Reg, RegState::ImplicitDefine);
+    }
```

Comment:
This is also missing an implicit use for any use instructions?

Reply:
The use of rounding control requires parsing a lot of instructions or just unconditional add use here. I prefer to not do so in this patch.